javascript/javascript3/week3/classmatesRepo/classmatesRepositories.js

```diff
@@ -0,0 +1,35 @@
+const urls = [
+  "https://api.github.com/search/repositories?q=user:nuha90",
+  "https://api.github.com/search/repositories?q=user:Elia-git",
+  "https://api.github.com/search/repositories?q=user:claudion3",
+];
+
+const body = document.querySelector("body");
+
+Promise.all(
+  urls.map((url) => {
+    return fetch(url).then((resp) => resp.json());
+  })
+).then((repos) => {
+  //console.log(repos);
+  repos.forEach((repo) => {
```

Comment:
Good
Try to make your code more readable by separating the behavior into functions that describe what you do. Words like li, body, ul1, ul2, repo and so on do not explain much. Always expect changes to come later. How might it change? and how might you prepare for those changes?